bndtools.core/src/bndtools/central/Central.java

```diff
@@ -204,39 +203,52 @@ public static Workspace getWorkspace() throws Exception {
             if (ws != null) { // early check for workspace
                 return ws;
             }
-            try {
-                Workspace.setDriver(Constants.BNDDRIVER_ECLIPSE);
-                Workspace.addGestalt(Constants.GESTALT_INTERACTIVE, new Attrs());
+            ws = initializeWorkspace();
+            // The workspace has been initialized fully, set the field now
+            workspace = ws;
+        }
+        workspaceQueue.resolve(ws); // notify onWorkspaceInit callbacks
```

Comment:
When I saw the changes to Central I was actually hoping that they render this PR obsolete :-)
I tested the SetupWorkspaceWizard on master but it didn´t work with an external location without restarting Eclipse. I will do another test since there have been some changes since the last time.

The main reason for this PR was issue #916 (I just found #1399 because I was looking for a working example)
I already have a working import ready, but it suffers the same problem as WorkspaceSetupWizard...you have to restart Eclipse to reset Central.

**If Central is not initialized with anything until there is a proper cnf project (in Eclipse), both issues should work without this PR.**

Waiting for #1430 is probably best. 
If you want to close this PR, feel free to do so. If necessary I will open a new one, assuming I found a proper solution with the workspace-invariant.